common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -48,6 +48,52 @@ class SequenceFields(object):
         scope=Scope.content,
     )
 
+    is_time_limited = Boolean(
+        display_name=_("Is Timed Limited"),
+        help=_(
+            "This field indicates if this courseware container will have a time limit to view/interact. "
```

Comment:
Instead of "courseware container" does "courseware component" work?
Suggest: "This setting indicates whether students have a limited time to view or interact with this courseware component."